### PR TITLE
Core welcome in new design

### PR DESCRIPTION
**Pull request by debugwand**

### CODE REVIEW

**Comment by laurieboyes** [comment]
Cool 👍

File: welcome-message/banner.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	<h6 class="n-welcome-banner__heading n-welcome-banner__column n-welcome-banner__column-m">Welcome to the new FT.com.</h6>
+
+	<div class="o-expander__content n-welcome-banner__column n-welcome-banner__column-fill">
+		<div class="n-welcome-banner__wrapper">
+			<p class="n-welcome-banner__column n-welcome-banner__column-xl n-welcome-banner__strapline">The same global insight. Faster than ever before on all your devices.</p>
+			<p class="n-welcome-banner__column n-welcome-banner__column-fill">
+				<a href="/myft/tour" class="n-welcome-banner__button" data-trackable="tour-page">Take the tour</a>
+			</p>
+			<div class="n-welcome-banner__column n-welcome-banner__column-actions">
+				<a href="#" data-trackable="help" class="n-welcome-banner__link-action">Help</a>
```
Comment (phamann) [inline welcome-message/banner.html]: href?

**Comment by phamann** [comment]
`n-welcome-banner__column-xl`
`n-welcome-banner__column-fill`
`n-welcome-banner__column-m`

Shouldn't these all be modifiers and not elements i.e. `n-welcome-banner__column n-welcome-banner__column--xl`?

**Comment by keirog** [comment]
Not sure I’m comfy with straying from the o-grid+flexbox conventions… Were they causing issues? Otherwise looks good

**Pull request merged**